internal/core/component.rs

```diff
@@ -82,6 +82,17 @@ pub struct ComponentVTable {
     /// strictly speaking not an Item at all!
     pub parent_node: extern "C" fn(core::pin::Pin<VRef<ComponentVTable>>, result: &mut ItemWeak),
 
+    /// This embeds a (tree of) components from one compilation unit into another
+    /// (tree of) components from a different compilation unit.
```

Comment:
Maybe something like that?

```suggestion
    /// This embeds this component into the item tree of another component
```

There is no concept of compilation unit right now. Especially not at runtime.